tests/queries/0_stateless/03632_lowcard_join.sql

```diff
@@ -0,0 +1,15 @@
+SET allow_suspicious_low_cardinality_types = 1;
+
+DROP TABLE IF EXISTS t0;
+DROP TABLE IF EXISTS t1;
+DROP TABLE IF EXISTS t2;
+
+CREATE TABLE t0 (x Int, y LowCardinality(Nullable(Int)) ALIAS x) ENGINE = MergeTree ORDER BY x;
+CREATE TABLE t1 (y LowCardinality(Int)) ENGINE = MergeTree ORDER BY y;
+CREATE TABLE t2 (y Nullable(Int)) ENGINE = MergeTree ORDER BY y SETTINGS allow_nullable_key = 1;
+
```

Comment:
Could you add a small comment explaining what the issue/error was for posterity?